### PR TITLE
Slow DNS

### PR DESCRIPTION
**Pull request by IvanRibakov**

### CODE REVIEW

File: sipplauncher/DnsServer.py [hunk 1]
```diff
@@ -185,7 +193,12 @@ def __load(run_id, file):
         Resolver.__get_logger(run_id).info('loading DNS file {0}'.format(file))
         records = []
         with open(file, 'r') as f:
+            options = {}
             for line in f:
```
Comment (IvanRibakov) [inline sipplauncher/DnsServer.py:197]:
```suggestion
            for line in f:
                options = {}
```

File: sipplauncher/DnsServer.py [hunk 2]
```diff
@@ -202,7 +215,8 @@ def __load(run_id, file):
                 else:
                     args = (args_,)
 
-                record = Record(rname, rtype, args)
+                record = Record(rname, rtype, args, options)
+                options = {}
```
Comment (IvanRibakov) [inline sipplauncher/DnsServer.py:219]:
```suggestion
```

**Comment by luismartingil** [comment]
@IvanRibakov @NGSegovia Do we want to merge this?

**Comment by IvanRibakov** [comment]
@luismartingil I think so. Let me bump sipplauncher version and I'll merge this PR.

File: sipplauncher/DnsServer.py (hunk 1 above)
Comment (NGSegovia) [reply sipplauncher/DnsServer.py:197]: Nope, that will break the logic, options will be used in next loop iteration (consider we are reading line by line, and options are applied to the next line).

**Review by NGSegovia — APPROVED** [review]

File: sipplauncher/DnsServer.py (hunk 1 above)
Comment (IvanRibakov) [reply sipplauncher/DnsServer.py:197]: > and options are applied to the next line

What do you mean?

```
#! {"delay": 10000}

// SRV record1

// A record2
```

Are you saying it's intended behaviour for delay to be applied to both `record1` and `record2` in this case? Not sure I'm liking this approach much. It doesn't seem very intuitive to have to "turn off" delay when it's not needed by setting 
```
#! {"delay": 0}
```

I don't see big problem with applying options to each individual record (if needed).

If repetition is **really** of concern, maybe we can think of some alternative marker (`#!!`, `#?` or any other) to specify "global" options once at the beginning of the file.

Comment (NGSegovia) [reply sipplauncher/DnsServer.py:197]: No, will apply only to record 1, this is why options is reset in line 219. 
https://github.com/zaleos/sipplauncher/pull/86/files#diff-cdc85d09a1322cede6e59df586825174bc3939d2296c6e27f3f0838507e0555aR219

Comment (IvanRibakov) [reply sipplauncher/DnsServer.py:197]: Sorry for the noise, big facepalm on my side.

**Pull request merged**